File: JavaFundamentals/src/BasicSyntaxConditionalStatementsAndLoops/Exercise/PrintAndSum.java
package BasicSyntaxConditionalStatementsAndLoops.Exercise;

import java.util.Scanner;

public class PrintAndSum {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int start = Integer.parseInt(scanner.nextLine());
        int end = Integer.parseInt(scanner.nextLine());

        int result = 0;
        for (int i = start; i <= end; i++) {
            System.out.printf("%d ", i);
            result += i;
        }
        System.out.print("\n");

        System.out.printf("Sum: %d", result);
    }
}

File: JS Front End/JS Syntax Fundamentals - Exercises/printAndSum.js
function printAndSum(start, end) {
    let result = 0;
    const numbers = [];
    for (let i = start; i <= end; i++) {
        result += i;
        numbers.push(i);
    }

    console.log(numbers.join(' '));
    console.log(`Sum: ${result}`);
}

printAndSum(5, 10);
printAndSum(0, 26);